### PR TITLE
Time Series generation refactor

### PR DESCRIPTION
**Pull request by payetvin**

### CODE REVIEW

File: .github/workflows/ubuntu.yml [hunk 1]
```diff
@@ -191,6 +191,15 @@ jobs:
...
 
+    - name: Run tests for time series generator tool
+      if: ${{ env.RUN_SIMPLE_TESTS == 'true' }}
+      uses: ./.github/workflows/run-tests
+      with:
+        simtest-tag: ${{steps.simtest-version.outputs.prop}}
+        batch-name: ts-generator
+        os: ${{ env.os }}
+        variant: "tsgenerator"
+
```
Comment (flomnes) [inline .github/workflows/ubuntu.yml]: revert

File: .github/workflows/windows-vcpkg.yml [hunk 2]
```diff
@@ -261,6 +261,15 @@ jobs:
...
 
+    - name: Run tests for time series generator tool
+      if: ${{ env.RUN_SIMPLE_TESTS == 'true' }}
+      uses: ./.github/workflows/run-tests
+      with:
+        simtest-tag: ${{steps.simtest-version.outputs.prop}}
+        batch-name: ts-generator
+        os: ${{ env.test-platform }}
+        variant: "tsgenerator"
+
```
Comment (flomnes) [inline .github/workflows/windows-vcpkg.yml]: revert

File: src/tools/ts-generator/main.cpp [hunk 3]
```diff
@@ -127,7 +125,7 @@ int main(int argc, char *argv[])
     }
 
     study->initializeRuntimeInfos();
-    // Force the writing of generated TS into output/YYYYMMDD-HHSSeco/ts-generator/thermal/mc-0
+    // Force the writing of generated TS into output/YYYYMMDD-HHSSeco/ts-generator/thermal
```
Comment (flomnes) [inline src/tools/ts-generator/main.cpp]: Revert comment to original (still mc-0)

**Review by flomnes — COMMENTED** [review]
Minor remarks

File: .github/workflows/ubuntu.yml (hunk 1 above)
Comment (payetvin) [reply .github/workflows/ubuntu.yml]: Tests for thermal ts generation, why remove it ?

File: src/tools/ts-generator/main.cpp (hunk 3 above)
Comment (payetvin) [reply src/tools/ts-generator/main.cpp]: It's not mc-0 if we generate thermal series through the tool

Comment (flomnes) [reply src/tools/ts-generator/main.cpp]: So then `output/YYYYMMDD-HHSSeco/ts-generator/thermal[/mc-0]`

File: .github/workflows/ubuntu.yml (hunk 1 above)
Comment (flomnes) [reply .github/workflows/ubuntu.yml]: Out of scope

**Review by flomnes — APPROVED** [review]

**Pull request merged**